srcdeps-core-config-yaml/src/test/java/org/srcdeps/config/yaml/YamlConfigurationIoTest.java

```diff
@@ -119,6 +121,76 @@ public void readFull() throws ConfigurationException, UnsupportedEncodingExcepti
         }
     }
 
+
+    @Test
+    public void readFull21Selectors() throws ConfigurationException, UnsupportedEncodingException, IOException {
+        try (Reader in = new InputStreamReader(getClass().getResourceAsStream("/srcdeps-full-2.1-selectors.yaml"), "utf-8")) {
+            Configuration actual = new YamlConfigurationIo().read(in).build();
+            Configuration expected = Configuration.builder() //
+                    .configModelVersion("2.1") //
+                    .forwardProperty("myProp1") //
+                    .forwardProperty("myProp2") //
+                    .builderIo(BuilderIo.builder().stdin("read:/path/to/input/file")
+                            .stdout("write:/path/to/output/file").stderr("err2out"))
+                    .skip(true) //
+                    .sourcesDirectory(Paths.get("/home/me/.m2/srcdeps")) //
+                    .verbosity(Verbosity.debug) //
+                    .buildTimeout(new Duration(35, TimeUnit.MINUTES)) //
+                    .maven( //
+                            Maven.builder() //
+                                    .versionsMavenPluginVersion("1.2") //
+                                    .failWith( //
+                                            MavenAssertions.failWithBuilder() //
+                                                    .addDefaults(false) //
+                                                    .goal("goal1") //
+                                                    .goal("goal2") //
+                                                    .profile("profile1") //
+                                                    .profile("profile2") //
+                                                    .property("property1") //
+                                                    .property("property2") //
+                            ) //
+                                    .failWithout( //
+                                            MavenAssertions.failWithoutBuilder() //
+                                                    .goal("goalA") //
+                                                    .goal("goalB") //
+                                                    .profile("profileA") //
+                                                    .profile("profileB") //
+                                                    .property("propertyA") //
+                                                    .property("propertyB") //
+                            ) //
+                    ) //
+                    .repository( //
+                            ScmRepository.builder() //
+                                    .id("org.repo1") //
+                                    .verbosity(Verbosity.trace) //
+                                    .include("group1") //
+                                    .include("group2:artifact2:*") //
```

Comment:
As I said, the change is backwards compatible, because `selectors` is kept as an alias of `includes`. This test asserts that the present parser can still correctly read the 2.1 files.